### PR TITLE
Update Example 9 to use Gazebo Sim and .xml launch files

### PR DESCRIPTION
**Pull request by destogl**
Used for testing ros-controls/ros2_control#1354.

@christophfroehlich, This should help you with #482—it should replace it if that is OK with you.

### CODE REVIEW

**Comment by christophfroehlich** [comment]
@destogl except for the world file, where is the difference to #482? And why does one need the world file? At least, I updated also the description.

File: example_9/bringup/launch/rrbot_gazebo_sim.launch.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    <param name="use_sim_time" value="true" />
+  </node>
+
+  <!-- Start Gazebo -->
+  <let name="gazebo_world" value="$(find-pkg-share ros2_control_demo_example_9)/worlds/empty.world" />
+  <include file="$(find-pkg-share ros_gz_sim)/launch/gz_sim.launch.py" >
+    <arg name="gz_args" value=" -r -v 4 $(var gazebo_world)"/>
+  </include>
+
+  <node pkg="ros_gz_bridge" exec="parameter_bridge" args="
```
Comment (olivier-stasse) [inline example_9/bringup/launch/rrbot_gazebo_sim.launch.xml:41]: Why not naming the node `clock_bridge` as in the python equivalent ?

File: example_9/bringup/launch/rrbot_gazebo_sim.launch.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  <node pkg="controller_manager" exec="spawner" args="forward_position_controller">
+    <param name="use_sim_time" value="true" />
+  </node>
+
+  <!-- Rviz -->
+  <node if="$(var gui)" pkg="rviz2" exec="rviz2" output="log" args="-d $(find-pkg-share $(var ros2_control_demo_description))/rrbot/rviz/rrbot.rviz">
+    <param name="use_sim_time" value="true" />
+  </node>
+
+</launch>
```
Comment (olivier-stasse) [inline example_9/bringup/launch/rrbot_gazebo_sim.launch.xml:64]: Dear @destogl maybe we do not need the equivalent xml file here, as it is not in the other examples.

File: example_9/worlds/empty.world [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+      name="gz::sim::systems::SceneBroadcaster">
+    </plugin>
+    <plugin
+      filename="gz-sim-contact-system"
+      name="gz::sim::systems::Contact">
+    </plugin>
+    <plugin
+      filename="gz-sim-sensors-system"
+      name="gz::sim::systems::Sensors">
+      <render_engine>ogre2</render_engine>
```
Comment (olivier-stasse) [inline example_9/worlds/empty.world:27]: It looks like this is the only difference with the default gz-sim `empty.world` file.

Do we need it here ?

**Review by olivier-stasse — COMMENTED** [review]
Dear @destogl and @christophfroehlich  thanks a lot for both contributions. Switching to gz-sim is definitely a plus. 

But keeping the examples simple and documented are the main target of the repo I guess.

**Comment by destogl** [comment]
> @destogl except for the world file, where is the difference to #482? And why does one need the world file? At least, I updated also the description.

@christophfroehlich you are right, I just tested and we don't need world file. I agree that we can go with #482, only thing I would like to add on top of it is `.launch.xml` as they are much more simpler.

**Pull request closed without merging**